graphql/schema/types/stash-box.graphql

```diff
@@ -13,11 +13,13 @@ input StashBoxInput {
 type StashID {
   endpoint: String!
   stash_id: String!
+  updated_at: Time!
 }
 
 input StashIDInput {
   endpoint: String!
   stash_id: String!
+  updated_at: Time!
```

Comment:
I think this needs to be optional to avoid a breaking change for clients. It should default to `now` if not provided.